### PR TITLE
unit tests for gwas.py

### LINKED ISSUE #116: Implement unit tests for gwas.py
Good to implement unit tests for gwas.py, following https://github.com/comorment/containers/blob/main/usecases/gwas_demo.md (but just the part that invokes gwas.py, not the part that executes the actual gwas)

### PR DESCRIPTION
**Pull request by ttfiliz**
Fixes #116 

Changes proposed in this pull request:

- Added unit tests for `gwas.py` based on the use case [gwas_demo](https://github.com/comorment/containers/blob/main/usecases/gwas_demo.md)

## Before submitting

- [x] I've read and followed all steps in the [Making a pull request](https://github.com/comorment/containers/blob/main/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [ ] I've updated or added any relevant docstrings following the syntax described in the
    [Writing docstrings](https://github.com/comorment/containers/blob/main/CONTRIBUTING.md#writing-docstrings) section of the `CONTRIBUTING` docs.
- [ ] If this PR fixes a bug, I've added a test that will fail without my fix.
- [ ] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

### CODE REVIEW

**Review by espenhgn — COMMENTED** [review]
Thanks @ttfiliz . I'm getting some test errors with this as`/REF/` dir doesn't exist from within the container w. `py.test -v tests/test_gwas_py.py`.
```
...
Traceback (most recent call last):
  File "/tmp/tmpxw0lxn6n/gwas.py", line 1005, in <module>
    args.func(args, log)
  File "/tmp/tmpxw0lxn6n/gwas.py", line 625, in execute_gwas
    fix_and_validate_pheno_args(args, log)
  File "/tmp/tmpxw0lxn6n/gwas.py", line 224, in fix_and_validate_pheno_args
    check_input_file(args.fam)
  File "/tmp/tmpxw0lxn6n/gwas.py", line 823, in check_input_file
    raise ValueError("Input file does not exist: {f}".format(f=fname))
ValueError: Input file does not exist: /REF/examples/regenie/example_3chr.fam
...
```
Can you make sure that you don't have host environment variables like `SINGULARITY_BIND` in effect? Setting this at the Python level explicitly within the test script ought to do I believe(`os.environ["SINGULARITY_BIND"] = ...`)

**Review by espenhgn — APPROVED** [review]
Thanks, think this is now good to go!

File: tests/test_gwas_py.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import subprocess
+import tempfile
+
+p3_sif = os.path.abspath(os.path.join("singularity", "python3.sif"))
+cwd = os.getcwd()
+ref = os.path.join(cwd, "reference", "examples", "regenie")
+
+os.environ["SINGULARITY_BIND"] = (
+    f"{os.path.join(cwd,'reference')}:/REF:ro,"
+    f" {os.path.join(cwd, '..', 'reference')}:/REF2:ro"
```
Comment (ofrei) [inline tests/test_gwas_py.py:17]: Best to drop `` f" {os.path.join(cwd, '..', 'reference')}:/REF2:ro"`` line, and avoid taking dependency on github.com/comorment/reference in unit-tests

**Comment by espenhgn** [comment]
@ofrei, can you merge this? I'm fine with it. All tests pass on my end.

**Pull request merged**